samples/edge_impulse/wrapper/sample.yaml

```diff
@@ -32,4 +32,4 @@ tests:
   sample.edge_impulse.wrapper:
     sysbuild: true
     build_only: false
-    tags: sysbuild
+    tags: sysbuild edge_impulse
```

Comment:
Should we add this tag to machine learning application?

Reply:
I want to take is separately - it is different folder.
I do not want to spread tag across folders tree.
Currently I add tags based on folders rather than looking at sample/tests which has similar dependencies.